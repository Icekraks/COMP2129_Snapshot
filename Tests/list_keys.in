set a 1 2 3
get a
set b 5 4 2
get b
list keys
bye
