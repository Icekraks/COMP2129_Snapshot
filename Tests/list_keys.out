ok
[1 2 3]
ok
[5 4 2]
b
a
bye
